services/src/main/java/org/keycloak/services/resources/account/AccountConsoleFactory.java

```diff
@@ -0,0 +1,44 @@
+package org.keycloak.services.resources.account;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import java.util.Map;
+import org.keycloak.Config.Scope;
+import org.keycloak.events.EventBuilder;
+import org.keycloak.models.ClientModel;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.KeycloakSessionFactory;
+import org.keycloak.models.ProtocolMapperModel;
+import org.keycloak.models.RealmModel;
+import org.keycloak.provider.ProviderEvent;
+import org.keycloak.services.resource.AccountResourceProvider;
+import org.keycloak.services.resource.AccountResourceProviderFactory;
+import org.keycloak.theme.Theme;
+import jakarta.ws.rs.NotFoundException;
+import org.keycloak.models.Constants;
+
+public class AccountConsoleFactory implements AccountResourceProviderFactory {
+  public static final String ID = "account-console";
+
+  @Override
+  public String getId() {
+    return ID;
```

Comment:
I would just use `default` to make it consistent with other default providers

```suggestion
    return ID;
```

Reply:
Set to`default`